src/test/resources/org/prebid/server/ApplicationTest/amp/test-cache-response.json

```diff
@@ -2,6 +2,9 @@
   "responses": [
     {
       "uuid": "fea00992-651c-44c8-b16a-b9af99fdf2dd"
+    },
```

Comment:
This file isn't used anymore, is it?

Some other cache response files also became obsolete as far as I understand, let's remove them as well